test-cases/longevity/longevity-large-collections-48h.yaml

```diff
@@ -0,0 +1,19 @@
+# 48 hours
+test_duration: 2880
+prepare_write_cmd: "cassandra-stress write cl=ALL n=1000000  -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..200200300 -log interval=15"
+stress_cmd: [
+"JVM_OPTION='-Xms8033M -Xmx8033M -Xmn400M' cassandra-stress user profile=/tmp/large_collections.yaml ops'(insert=3, read1=1)' cl=QUORUM duration=2860m -port jmx=6868 -mode cql3 native -rate threads=10"
```

Comment:
Did you really need to increase the JVM memory when lowering the size of list to 500 and 1000?

Reply:
CS will fail for OUT of Memory if we don't increase the memory of JVM, even the size of list is 500.

I reduced the size of list to avoid the c-s stuck, not out-of-memory issue.